nixos/modules/programs/nh.nix

```diff
@@ -0,0 +1,93 @@
+{ config
+, lib
+, pkgs
+, ...
+}: {
+  meta.maintainers = [ lib.maintainers.viperML ];
+
+  options.programs.nh = {
+    enable = lib.mkEnableOption (lib.mdDoc "Yet another Nix CLI helper.");
```

Comment:
`lib.mdDoc` is a no-op

Reply:
also we should remove the trailing period when using `mkEnableOption`

Reply:
Shouldn't all module descriptions include mdDoc ?